ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/port-channel-interfaces.yml

```
@@ -3,6 +3,9 @@
 port_channel_interfaces:
   - name: Port-Channel5
     description: DC1_L2LEAF1_Po1
+    comment: |-
+      testing multi line comments with | -
```

Should it be like this?
```suggestion
      testing multi line comments with |-
```

---

I think the extra `| - ` is a typo!